identity/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.3
+
+- Update `ed25519-dalek` to `2.0` [PR 4337]
```

Comment:
```suggestion
- Update `ed25519-dalek` to `2.0`.
  See [PR 4337].
```

Normally we don't make entries for dependency bumps but we can't release this otherwise.

Reply:
Can we change this be a "fix" for the RUSTSEC vulnerability? Like `Fix RUSTSEC-XYZ by updating to `ed25519-dalek 2.0` or something.

Reply:
yup, updated :)